src/iocore/dns/unit_tests/test_SplitDNSYAMLLoader.cc

```diff
@@ -0,0 +1,82 @@
+/** @file
+
+  Catch based unit tests for SplitDNS
+
+  @section license License
+
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements.  See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership.  The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+
+#include "iocore/dns/SplitDNSYAMLLoader.h"
+#include "P_SplitDNSProcessor.h"
+
+#include <catch.hpp>
+
+#include <sstream>
+#include <string_view>
+
+static bool
+error_contains(std::string_view error, std::string_view substr)
+{
+  return error.find(substr) != std::string_view::npos;
```

Comment:
Why not just use [`std::string_view::contains`](https://en.cppreference.com/w/cpp/string/basic_string_view/contains)?

Reply:
Isn't that a C++23 feature?

Reply:
Oops! You're correct. I wonder if I should put it in `TextView`.